app/assets/stylesheets/custom.css.scss

```diff
@@ -75,66 +75,73 @@ h1 {
 #flash_notice {
   font-family: monospace;
   text-style: bold;
-  text-color: red;
   opacity: .8;
   text-align: center;
   margin-right: 60%;
   padding:4px;
   border-radius: 100px;
-  color: cornsilk;
+  color: #fff8dc;
   background-color: darkgrey;
 }
 
 #flash_errors{
   font-family: monospace;
   text-style: bold;
-  text-color: red;
   opacity: .8;
   text-align: center;
   margin-right: 60%;
   padding:4px;
   border-radius: 100px;
-  color: cornsilk;
+  color: #fff8dc;
   background-color: darkgrey;
 }
 
 #flash_messages{
   font-family: monospace;
   text-style: bold;
-  text-color: red;
   opacity: .8;
   text-align: center;
   margin-right: 60%;
   padding:4px;
   border-radius: 100px;
-  color: cornsilk;
+  color: #fff8dc;
   background-color: darkgrey;
 }
 #flash_message{
   font-family: monospace;
   text-style: bold;
-  text-color: red;
   opacity: .8;
   text-align: center;
   margin-right: 60%;
   padding:4px;
   border-radius: 100px;
-  color: cornsilk;
+  color: #fff8dc;
   background-color: darkgrey;
 }
 #flash_error{
   font-family: monospace;
   text-style: bold;
-  text-color: red;
   opacity: .8;
   text-align: center;
   margin-right: 60%;
   padding:4px;
   border-radius: 100px;
-  color: cornsilk;
+  color: #fff8dc;
   background-color: darkgrey;
 }
 
+#flash_twitter {
+  font-family: monospace;
+  text-style: bold;
+  opacity: .8;
+  text-align: center;
+  margin-right: 60%;
+  padding:4px;
+  border-radius: 100px;
+  color: #fff8dc;
+  background-color: #ff0000;
```

Comment:
Color literals like `#ff0000` should only be used in variable declarations; they should be referred to via variable everywhere else.